tests/ft_cargo.vader

```diff
@@ -0,0 +1,21 @@
+Include: include/setup.vader
```

Comment:
Not sure about the `ft_` prefix here, since it's not a filetype maker after all.
Rename it to `cargo.vader`?